OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -181,7 +181,19 @@ bool TracerThread::OpenSampling(const std::vector<int32_t>& cpus) {
   std::vector<int> sampling_tracing_fds;
   std::vector<PerfEventRingBuffer> sampling_ring_buffers;
   for (int32_t cpu : cpus) {
-    int sampling_fd = sample_event_open(sampling_period_ns_, -1, cpu);
+    int sampling_fd;
+    switch (tracing_options_.unwinding_method_) {
+      case kFramePointers:
+        sampling_fd = sample_event_callchain_open(sampling_period_ns_,
+                                                  -1,
+                                                  cpu);
+        break;
+      case kDwarf:sampling_fd = sample_event_open(sampling_period_ns_, -1, cpu);
+        break;
+      case kUndefined:ERROR("Undefined Unwinding Method");
```

Comment:
The format is quite broken here.

Reply:
Thanks for spotting this. I'll run clang-format when I'm done with addressing the review comments.